src/test/kotlin/blackjack/model/BetTest.kt

```diff
@@ -0,0 +1,59 @@
+package blackjack.model
+
+import blackjack.utils.Constants
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+class BetTest {
+    @Test
+    fun `player is created with a name and bet`() {
+        val bet = Bet(10000)
+        val player = Player("player", bet)
+
+        assertEquals("player", player.name)
+        assertEquals(10000, player.bet.amount)
+    }
+
+    @Test
+    fun `bet should be positive`() {
+        assertThrows<IllegalArgumentException> {
+            Bet(-100)
+        }
+    }
+
+    @Test
+    fun `bet should be more than 0`() {
+        assertThrows<IllegalArgumentException> {
+            Bet(0)
+        }
+    }
```

Comment:
it's not important 😄, but 0 is not positive!
how about this?

"bet should not be negative" `Bet(-1)`